File: src/main/java/com/github/njuro/jard/config/security/NoRedirectStrategy.java
package com.github.njuro.jard.config.security;

import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;
import org.springframework.security.web.RedirectStrategy;

/** No-op redirect strategy. */
public class NoRedirectStrategy implements RedirectStrategy {

  @Override
  public void sendRedirect(HttpServletRequest request, HttpServletResponse response, String url) {
    // no redirect
  }
}

File: src/main/kotlin/com/github/njuro/jard/rewrite/config/security/NoRedirectStrategy.kt
package com.github.njuro.jard.rewrite.config.security

import org.springframework.security.web.RedirectStrategy
import javax.servlet.http.HttpServletRequest
import javax.servlet.http.HttpServletResponse

/** No-op redirect strategy.  */
class NoRedirectStrategy : RedirectStrategy {
    override fun sendRedirect(request: HttpServletRequest, response: HttpServletResponse, url: String) {
        // no redirect
    }
}